simuleval/data/dataloader/t2t_dataloader.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+from pathlib import Path
+from typing import Callable, List, Union, Optional
+from .dataloader import GenericDataloader
+from simuleval.data.dataloader import register_dataloader
+from argparse import Namespace
+
+
+@register_dataloader("text-to-text")
+class TextToTextDataloader(GenericDataloader):
+    def __init__(
+        self, source_list: List[str], target_list: List[Optional[str]]
+    ) -> None:
+        super().__init__(source_list, target_list)
+        self.source_splitter = lambda x: x.split()
+        self.target_splitter = lambda x: x
```

Comment:
No here because we don't need to pass the target to agent. However, there should be better design on preprocess the source and target text. For instance current version haven't consider languages without space, such as Chinese and Japanese. I make an update soon.